deploy/examples/ApplicationMonitoring.yaml

```diff
@@ -4,3 +4,7 @@ metadata:
   name: example-applicationmonitoring
 spec:
   labelSelector: "middleware"
+  blackboxTargets:
+    - service: nexus
+      url: http://nexus.nexus:8081/nexus/content/groups/public
```

Comment:
Would the exposed alertmanager route be better as an example as it would exist with an out of the box install, and would be an easy way to test the probing i.e. deleting/scaling the alertmanager pods and seeing the failing probe?

Reply:
Updated to use `https://example.com`.